app/templates/index.html

```diff
@@ -78,6 +78,13 @@ <h3 class="heading-xmedium">Buyers</h3>
       </p>
   {% endif %}
 
+  {% if current_user.has_any_role('admin-ccs-category', 'admin-framework-manager', 'admin-ccs-sourcing') %}
+      <h2 class="heading-large">Outcomes</h2>
+      <p>
+        <a href="{{ url_for('.download_outcomes') }}">Download G-Cloud outcomes</a>
```

Comment:
Can it just be 'Direct Award' outcomes? We want to avoid mentioning specific frameworks if we can. Even tho framework independence is still a bit of a fantasy :)